script.js

```diff
@@ -19,39 +19,42 @@ recipeApp.getRecipe = () => {
             }
         }).then((res) => {
             for (let i = 0; i < 3; i++) {
-                console.log("this is length of needed array", res.meals.length);
+                // get three random dishes and limiting output on page set to only 3

```

Comment:
we better not use random number here, it's because for user experience, they might want to see the same recipe generated at first time again, like maybe they want to pull the same recipe a number of times. so I suggested to always return first three results after submit. we could do other suggestions with second click maybe for stretch goal. so here I used: res.meals.slice(0, 3)